### PR TITLE
images: Reduce libpwquality checks on Fedora images

### LINKED ISSUE #44: Image refresh for fedora-31
Image refresh for fedora-31
 * [ ] FAIL: image-refresh fedora-31

### PR DESCRIPTION
**Pull request by martinpitt**
libpwquality 1.4.1 now also applies to `chpasswd`, breaking a lot of our
tests that want to set our standard "foobar" password. Reduce the
requirements, as that password is baked into our CI in so many places.

Fixes #44

 * [x] image-refresh fedora-31

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 1-ci-srv-02.
Log: https://209.132.184.41:8493/logs/image-refresh-57-20190930-103535/

**Comment by cockpituous** [comment]
Task failed: https://209.132.184.41:8493/logs/image-refresh-57-20190930-103535/

File: images/scripts/fedora.setup [hunk 1]
```diff
@@ -22,6 +22,9 @@ if [ "$IMAGE" != fedora-31 ]; then
 fi
 df -h /
 
+# make libpwquality less aggressive, so that our "foobar" password works
+printf 'dictcheck = 0\nminlen = 8\n' >> /etc/security/pwquality.conf
```
Comment (marusak) [inline images/scripts/fedora.setup]: still fails the same, maybe because `len("foobar")=6`?

Comment (martinpitt) [reply images/scripts/fedora.setup]: Yeah, of course that was meant to be a 6.. Fixing in #48 first

**Comment by cockpituous** [comment]
image-refresh in progress on 3-ci-srv-05.
Log: https://209.132.184.41:8493/logs/image-refresh-57-20190930-120000/

**Comment by cockpituous** [comment]
image-refresh fedora-31 done: https://github.com/cockpituous/bots/commits/image-refresh-fedora-31-20190930-122626

**Comment by martinpitt** [comment]
[testNegotiate failure](https://209.132.184.41:8493/logs/pull-57-20190930-151313-d9156c63-cockpit-project-cockpit--fedora-31/log.html#174-2), this needs to be investigated.

**Comment by martinpitt** [comment]
testNegotiate: This seems to be a new regression in Fedora 31 in sssd 2.2.2. I reported it as https://bugzilla.redhat.com/show_bug.cgi?id=1757299

**Comment by martinpitt** [comment]
Added a naughty.

**Pull request merged**